### PR TITLE
Set up coveralls 

### PR DESCRIPTION
**Pull request by SophieSjoberg**
## PT link https://www.pivotaltracker.com/story/show/154143585

Set up Coveralls for testing environment

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -1,58 +1,17 @@
...
-#
-
-# Possible values are :truncation and :transaction
-# The :transaction strategy is faster, but might give you threading problems.
-# See https://github.com/cucumber/cucumber-rails/blob/master/features/choose_javascript_database_strategy.feature
 Cucumber::Rails::Database.javascript_strategy = :truncation
 
+Warden.test_mode!
+World Warden::Test::Helpers
+After { Warden.test_reset! }
```
Comment (tochman) [inline features/support/env.rb]: Setting up warden feels premature. Do you use Device yet?

Comment (NegativeKarma) [reply features/support/env.rb]: We installed the Devise gem, but we're not going to use it for the MVP

**Review by NegativeKarma — COMMENTED** [review]
It's all good, let's just decide about Warden

File: features/support/env.rb (hunk 1 above)
Comment (diraulo) [reply features/support/env.rb]: Why install it if you're not going to use it? Only install things you actually need

**Review by diraulo — COMMENTED** [review]
Coverage setup looks fine, I Just don't see any of the new gems added for that? Or was it done in a previous PR? 🤔

File: features/support/env.rb (hunk 1 above)
Comment (SophieSjoberg) [reply features/support/env.rb]: We removed it! :)

**Comment by SophieSjoberg** [comment]
We set up coverage during the initial environment set up, so the gems are there.

**Review by diraulo — APPROVED** [review]
Update the title and this can go.

**Comment by diraulo** [comment]
One more thing... make sure you run bundle again and checkin the updated Gemfile.lock

**Pull request merged**